app/test/e2e/utils.js

```diff
@@ -222,10 +222,47 @@ const mockLayer = (id = undefined, extraData = {}) => {
     return mockData;
 };
 
+const mockPostGraphAssocition = (id, mockSuccess = true) => {
+    nock(process.env.CT_URL)
+        .post(`/v1/graph/dataset/${id}/associate`)
+        .reply(mockSuccess ? 200 : 404, { data: {} });
+};
+
+const mockPutGraphAssocition = (id, mockSuccess = true) => {
```

Comment:
no need to change this now, but something to keep in mind fo the future: don't name stuff `id` unless it's super clear what id it is.
`Vocabulary.get(id)` -> pretty safe to assume it's a vocabulary id
`mockPutGraphAssocition(id)` -> id of what? the association? the vocabulary?

what you DO need to change is fix the typo in the function name. Add a spellchecker to your IDE please

Reply:
Clarified the parameter name of mock association functions (it's supposed to be the `datasetId`), and fixed typos 😅